transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/specification/ws/acceptor/BaseFramingIT.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.kaazing.gateway.transport.wsn.specification.ws;
+package org.kaazing.gateway.transport.wsn.specification.ws.acceptor;
 
 import static org.kaazing.test.util.ITUtil.createRuleChain;
```

Comment:
What does this utility method do?
Wouldn't it be more readable to have the usual `RuleChain.outerRule(...).around(...)` ?
Recommend minimal additional dependencies in test code to maximize readability of how it's really testing.